Form/Type/StaticContentType.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Form\Type;
+
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
+use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class StaticContentType extends AbstractType
+{
+    /**
+     * Builds the form.
+     *
+     * @see FormTypeInterface::buildForm()
+     *
+     * Available options:
+     *     - use_ivory_ckeditor bool use CKEditorType instead of TextareaType for body
+     *     - ivory_ckeditor_options array CKEditorType options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('title', TextType::class)
+            ->add(
+                'body',
+                $options['use_ivory_ckeditor'] ? CKEditorType::class : TextareaType::class,
+                $options['ivory_ckeditor_options']
```

Comment:
In https://github.com/symfony-cmf/content-bundle/blob/master/DependencyInjection/CmfContentExtension.php#L40-L66, we could instead conditionally register the FormExtension.

So the user will still setup ivory through the ContentBundle configuration (https://github.com/symfony-cmf/content-bundle/blob/master/DependencyInjection/Configuration.php#L45-L57).

With such setup, the admin will have no knowledge (and don't need) about type used for the body field.